### DISCUSSION TITLE
How to only use rich to print out colors specified in the code?

### DISCUSSION THREAD

**Discussion by avocardio** [Textualize/rich#1229] [Q&A] [answered]
Hello, 

I'm trying to only use colors for strings I specified in the code, which works, but I don't want the new colourful terminal or anything else. Is there any way to do this?

**Comment by willmcgugan** [comment, marked as answer]
I'm not certain I follow. I'm going to assume that by "new colorful terminal" you are referring to highlighting. See the [docs](https://rich.readthedocs.io/en/latest/highlighting.html) for how to disable it.

**Reply by avocardio** [reply]
Yes, thank you.
